src/fw-logs/fw-logs-formatting-options.cpp

```diff
@@ -8,6 +8,7 @@
 #include <rsutils/string/from.h>
 
 #include <sstream>
+#include <stdint.h>
```

Comment:
Hi @rurban, is the include needed here? It is already included in the h file

Reply:
Technically not. But the cpp uses it in uint32_t thread_id, so I added it here also.